gnuradio-runtime/include/gnuradio/fxpt.h

```diff
@@ -45,15 +45,16 @@ namespace gr {
     static const int NBITS = 10;
     static const float s_sine_table[1 << NBITS][2];
     static const float PI;
+    static const float TAU;
     static const float TWO_TO_THE_31;
 
   public:
     static int32_t
       float_to_fixed(float x)
     {
       // Fold x into -PI to PI.
-      int d = (int)floor(x/2/PI+0.5);
-      x -= d*2*PI;
+      int d = (int)floorf(x/TAU+0.5);
```

Comment:
I'm wondering if `std::floor` isn't essentially the same or better?

Reply:
well, pretty much everywhere else I used `std::<math function>`. Here I didn't; had a reason, but it was yesterday night and I can't remember. Lemmechecksomething.